edx_notifications/server/web/templates/django/notifications_widget.html

```diff
@@ -11,12 +11,37 @@
 
 <script src="{{ STATIC_URL }}edx_notifications/js/models/counter_icon_model.js"></script>
 <script src="{{ STATIC_URL }}edx_notifications/js/models/user_notification_model.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/models/notification_preferences_model.js"></script>
 <script src="{{ STATIC_URL }}edx_notifications/js/collections/notification_collection.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/collections/notification_preferences_collection.js"></script>
 
 <script src="{{ STATIC_URL }}edx_notifications/js/views/counter_icon_view.js"></script>
 <script src="{{ STATIC_URL }}edx_notifications/js/views/notification_pane_view.js"></script>
+<script src="{{ STATIC_URL }}edx_notifications/js/views/notification_preferences_view.js"></script>
 
-
+<script type="text/template" id="xns-notification-preferences-template">
+    <h2>Notification Preferences</h2>
```

Comment:
can you add a div wrapper around this with a class name, so we can apply targeted styling?